src/json-editor.component.scss

```diff
@@ -67,6 +67,10 @@ table {
     padding-right: 5px;
   }
 
+  td.error {
+    background-color: #f2968d !important;
```

Comment:
I am not sure if it's OK but maybe it's better to have `.error` instead of `td.error`, I feel like we will use it more